### PR TITLE
Adding health markdown document

### PR DESCRIPTION
**Pull request by stjohnjohnson**

### CODE REVIEW

**Comment by Filbird** [comment]
This is nice :+1

**Comment by nicolaifsf** [comment]
lgtm 👍

**Comment by nicolaifsf** [comment]
404 on the config parser?

**Comment by nkatzman** [comment]
This is cool. Is there any way to do this programattically ?

File: HEALTH.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+[dydb-veri]: https://img.shields.io/npm/v/screwdriver-datastore-dynamodb.svg
+[dydb-veru]: https://npmjs.org/package/screwdriver-datastore-dynamodb
+[dydb-bldi]: https://app.wercker.com/status/bebcdc9de9d33dc7dea39e388efec0c0
+[dydb-bldu]: https://app.wercker.com/project/bykey/bebcdc9de9d33dc7dea39e388efec0c0
+[dydb-depi]: https://david-dm.org/screwdriver-cd/datastore-dynamodb.svg?theme=shields.io
+[dydb-depu]: https://david-dm.org/screwdriver-cd/datastore-dynamodb
+
+[cfg-veri]: https://img.shields.io/npm/v/screwdriver-config-parser.svg
+[cfg-veru]: https://npmjs.org/package/screwdriver-config-parser
+[cfg-bldi]: https://app.wercker.com/status/1a0ecd0b5c31c377926531a27fe2e4dc
```
Comment (nkatzman) [inline HEALTH.md]: https://app.wercker.com/status/557eccbc3bebaca70f03b5093e3222f9

**Comment by stjohnjohnson** [comment]
I wish there was an easy way, I just wrote this last night :(

**Comment by d2lam** [comment]
👍

**Pull request merged**